.github/workflows/integration-test.yml

```diff
@@ -38,8 +38,7 @@ jobs:
       - name: Checkout repo
         uses: actions/checkout@v4
         with:
-          # Full history likely required for planned commitizen checks
-          # See #345
+          # Full history required for branch-based pre-commit checks
```

Comment:
Would it make sense to retain the mention about commitizen or dynamic versioning here (as a way of being explicit about why the line below is necessary)?

Reply:
Thanks, makes sense!